src/interface.jl

```diff
@@ -164,25 +167,29 @@ function _clear_f!(d::NLSolversBase.AbstractObjective)
         d.F = typeof(d.F)(NaN)
     end
     d.x_f .= eltype(d.x_f)(NaN)
+    d.x_f
```

Comment:
Is there a change in 0.7 that returns the right hand side of the dot-assignment if it's the last line?

Reply:
Yup